test/integration/test_navbar_links.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Checks that the links in the navbar on the homepage have the right URLs to redirect to
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows
+ */
+
+require('chromedriver');
```

Comment:
It's necessary as far as I can tell. We have to pass the chromedriver to selenium webdriver. I thought we weren't going to make testing packages part of package.json > dev dependencies?